modules/kamon/src/main/resources/kamon.conf

```diff
@@ -0,0 +1,43 @@
+kamon {
```

Comment:
this file needs to be added within a kamon module (which is not currently defined).